### PR TITLE
Homework completed

### PR DESCRIPTION
**Pull request by andstartut**
Homework for lesson2

### CODE REVIEW

File: Lesson2/src/Homework.java [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+        System.out.println(max(new int[]{1, 2, 3, 4, 5, 100, 99}));
+        System.out.println(calculateHypotenuse());
+    }
+
+    /**
+     * 1. Method should return sum of "a" and "b"
+     * 2. (optional) if sum of a and b is more then int can store -- return -1;
+     **/
+    public static int sum(int a, int b) {
+        return (a + b < 0 ? -1 : a + b);
```
Comment (GlebLitvinov) [inline Lesson2/src/Homework.java]: if a = -10, b = -20 then expected result will be -30, but this method will return -1
Good use of ternary operator though)

File: Lesson2/src/Homework.java [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+     * b = 5
+     * <p>
+     * Method should return 5
+     * Example2:
+     * a = 10,
+     * b = 10
+     * <p>
+     * Method should return 10
+     */
+    public static int max(int a, int b) {
```
Comment (GlebLitvinov) [inline Lesson2/src/Homework.java:36]: great

File: Lesson2/src/Homework.java [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+     * <p>
+     * Example2:
+     * Console input:
+     * 12
+     * 16
+     * Method should return 20
+     */
+    public static double calculateHypotenuse() {
+        Scanner consoleIn = new Scanner(System.in);
+        return Math.hypot(Double.valueOf(consoleIn.next()), Double.valueOf(consoleIn.next()));
```
Comment (GlebLitvinov) [inline Lesson2/src/Homework.java]: Good job using Math.hypot!
you can call consoleIn.nextDouble() instead of Double.valueOf(consoleIn.next()), it will take less space and will perform less operations

**Review by GlebLitvinov — CHANGES_REQUESTED** [review]

File: Lesson2/src/Homework.java (hunk 3 above)
Comment (andstartut) [reply Lesson2/src/Homework.java]: OK. I will try to be more attentive.

File: Lesson2/src/Homework.java (hunk 1 above)
Comment (andstartut) [reply Lesson2/src/Homework.java]: Somebody in the chat wrote that condition is doesn't change, and we won't use -10 and -20.)) I thought that I need to use only integer type.

Comment (GlebLitvinov) [reply Lesson2/src/Homework.java]: int range is from -2147483648 to 2147483647, so that's true, we use only integer type

File: .gitignore [hunk 4]
```diff
@@ -1,27 +0,0 @@
-#Intellij IDEA
```
Comment (GlebLitvinov) [inline .gitignore]: why did you delete this file and Lesson1.java?
